### ISSUE TITLE
Reimplement results not good as expected

### ISSUE THREAD

**Issue by Nerdary** [chuangchuangtan/FreqNet-DeepfakeDetection#2]
I clone the code and do the same as readme indicates, but the reimplement results are not as good as the released pretrained model, here are some details how I train the model.

First I test the result of released pretrained weights (the second row), then I fix the random seed and use the exactly same command in readme, however the results are not good as expected (the third row).

![image](https://github.com/chuangchuangtan/FreqNet-DeepfakeDetection/assets/37006358/ec2b170f-91c5-4648-86cf-0b75c99dfe41)

I download the train/val/test set of ForenSynths and place them in a fold named "ForenSynths“ to make sure the released codes can load data successfully.

I also modify the "test_model" function to directly test the model on test set an the end of each epoch, which is not supposed to influence the results.

Is there any solutions to fix this issue?

**Comment by chuangchuangtan** [comment]
Without setting a random seed, we cannot guarantee that the model generated in each training session will be the same. 
Additionally, there are unstable situations in frequency domain learning. 
We will soon release training parameters with random seed settings.

**Comment by TeAMoooov** [comment]
> Without setting a random seed, we cannot guarantee that the model generated in each training session will be the same. Additionally, there are unstable situations in frequency domain learning. We will soon release training parameters with random seed settings.

Hello , I also encountered the same problem. When will the training parameters with random seed settings be released?
Thank you very much.

**Comment by TeAMoooov** [comment]
您好，我也出现了复现精度上的问题，请问一下最近有考虑公布随机种子嘛？我尝试过使用代码中注释部分的种子，还是无法达到论文中的精度，谢谢您。

**Comment by wasim004** [comment]
Have the same issue when training for 1 class (horse)...
![Screenshot from 2024-09-03 11-58-17](https://github.com/user-attachments/assets/58447ea6-ca5a-4dd9-a167-cbc162c1788b)

**Comment by wasim004** [comment]
Also after training the model only on 1-class(Horse) when I ran the `test.py` the results are much weird. I had follow all the guidelines for training the model on the data mentioned in the section `Datasets-->Training Set` and `Datasets-->Real-world Scene Test set.` and finally got the following results.

![Screenshot from 2024-09-03 12-18-02](https://github.com/user-attachments/assets/c269cc41-b9e7-481d-8d36-d6052c446541)

**Comment by zgs-df** [comment]
> 您好，我也出现了复现精度上的问题，请问一下最近有考虑公布随机种子嘛？我尝试过使用代码中注释部分的种子，还是无法达到论文中的精度，谢谢您。

我也一样，太难了

**Comment by aqqwqq** [comment]
您好，请问有解决方案吗，我也出现了上述的问题
